components/gitpod-db/go/dbtest/oidc_client_config.go

```diff
@@ -31,10 +31,12 @@ func NewOIDCClientConfig(t *testing.T, record db.OIDCClientConfig) db.OIDCClient
 
 	now := time.Now().UTC().Truncate(time.Millisecond)
 	result := db.OIDCClientConfig{
-		ID:           uuid.New(),
-		Issuer:       "issuer",
-		Data:         encrypted,
-		LastModified: now,
+		ID:             uuid.New(),
+		OrganizationID: uuid.New(),
+		Issuer:         "issuer",
```

Comment:
[nit] Using a more realistic value may be better, as it helps set expectation what you may expect as value here